qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/InitialSetupSteps.java

```diff
@@ -15,6 +15,9 @@
 import org.junit.Assert;
 import org.openmrs.contrib.qaframework.helper.InitialSetupPage;
 import org.openmrs.contrib.qaframework.helper.TestBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
```

Comment:
Is this exception in use?

Reply:
> Is this exception in use?

Have done the needful in the next subsequent commit with id `9779260`